core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -245,13 +245,13 @@ class BasicWorkflow extends EnsimeSpec
           )
 
           // expand selection around 'val foo'
-          project ! ExpandSelectionReq(fooFile, 215, 215)
+          project ! ExpandSelectionReq(fooFile, 290, 290)
```

Comment:
Yeah, that's a bummer. I think scala-refactoring has some code to reliably find positions of names (for instance, for Rename). Otherwise could also handle this "on site" (same goes for all definitions, I imagine).